p22/tests/integration_test.rs

```diff
@@ -0,0 +1,101 @@
+use p22::fibonacci::{fibonacci_non_recursive, fibonacci_recursive};
+
+#[cfg(test)]
+// Source: https://oeis.org/A000045
+static FIBONACCI_NUMBERS: [u64; 41] = [
+    0,
+    1,
+    1,
+    2,
+    3,
+    5,
+    8,
+    13,
+    21,
+    34,
+    55,
+    89,
+    144,
+    233,
+    377,
+    610,
+    987,
+    1597,
+    2584,
+    4181,
+    6765,
+    10946,
+    17711,
+    28657,
+    46368,
+    75025,
+    121393,
+    196418,
+    317811,
+    514229,
+    832040,
+    1346269,
+    2178309,
+    3524578,
+    5702887,
+    9227465,
+    14930352,
+    24157817,
+    39088169,
+    63245986,
+    102334155u64,
+];
+
+#[cfg(test)]
```

Comment:
You don't need to use `#[cfg(test)]` in integration tests, since they are already compiled only in the test mode. If you want to keep test function separately, you can add separate `tests/fibonacci_recursive.rs` and  `tests/fibonacci_loop.rs` files.

Reply:
done